queries.sql

```diff
@@ -62,12 +71,15 @@ SELECT * FROM invoices ORDER BY total LIMIT 10;
 SELECT * FROM invoices WHERE billing_city = 'Chicago' AND billing_state = 'IL' AND billing_country='USA';
 
 -- A list of all the invoices worth more than $5.00 from Chicago, IL
+SELECT * FROM invoices WHERE billing_city = 'Chicago' AND total > 5;
 
 -- The billing addresses of the 5 most valuable invoices from Mountain View CA
 -- Gotta reward those big spenders!
+SELECT * FROM invoices WHERE billing_city = 'Mountain View' ORDER BY total DESC LIMIT 5;
 
 -- A list of the 10 most valuable invoices made before January 1st, 2010
 -- Hint: Dates are formatted like 'YYYY-MM-DD' and you can compare them using '<', '>', '<=' and '>='
+SELECT * FROM invoices WHERE invoice_date <= '2010-01-01' ORDER BY total DESC LIMIT 10;
```

Comment:
This will select every on or before January 1st, 2010. If you only want invoices before January 1st, 2020, you'd use `<`.

But also...you use `<` here. Why not go back up and fix this query: https://github.com/kross00/sql-queries/pull/1/files#diff-f836187b3cdddbb192505dc1204ef225R45